en/_mdb5/standard/web/docs/cli/reference/o.html

```diff
@@ -87,6 +87,30 @@ <h2 class="mb-4">mdb version|-v|--version</h2>
 
   <hr class="my-5" />
 
+  <section id="section-register">
+    <!-- Section title -->
+    <h2 class="mb-4">mdb register</h2>
+
+    <p>
+      Create your MDB account. It will ask you to provide the username, email and password.
```

Comment:
I think we should also point out that the user will be automatically logged in so there's no need to run `mdb login` additionally.